commands.js

```diff
@@ -2476,8 +2476,11 @@ exports.commands = {
 		if (!this.can('hotpatch')) return false;
 		if (Monitor.hotpatchLock) return this.errorReply("Hotpatch is currently been disabled. (" + Monitor.hotpatchLock + ")");
 
-		let staff = Rooms('staff');
-		if (staff) staff.add("(" + user.name + " used /hotpatch " + target + ")").update();
+		let announceRooms = ['development', 'staff', 'upperstaff'];
+		for (let i in announceRooms) {
```

Comment:
I'll do it myself.

I guess I didn't write it in since I assumed everyone knew not to use `for..in` for arrays.